### PR TITLE
TestSecureController.java for Liberty needs root context based on artifactId

### LINKED ISSUE #209: Liberty doesn't work if artifactId is not "demo" due to hardcoded context
[src/main/resources/files/liberty/TestSecureController.java.tpl#L51](https://github.com/eclipse/microprofile-starter/blob/master/src/main/resources/files/liberty/TestSecureController.java.tpl#L51)

### PR DESCRIPTION
**Pull request by rdebusscher**
Fixes #209.

### CODE REVIEW

File: src/main/resources/files/liberty/TestSecureController.java.tpl [hunk 1]
```diff
@@ -48,7 +48,7 @@ public class TestSecureController {
         String jwt = generateJWT(key);
 
         // any method to send a REST request with an appropriate header will work of course.
-        WebTarget target = ClientBuilder.newClient().target("http://localhost:8281/demo/data/protected");
+        WebTarget target = ClientBuilder.newClient().target("http://localhost:8281/[# th:text="${artifact_id}"/]/data/protected");
```
Comment (Karm) [inline src/main/resources/files/liberty/TestSecureController.java.tpl]: @rdebusscher Cannot we parameterize the URL and have just one TestSecureController template? IMHO this is te only LIberty specific line, right...?

Comment (rdebusscher) [reply src/main/resources/files/liberty/TestSecureController.java.tpl]: Or we can have a look at how we deploy app on context root with Liberty.  That would be easier ....

Comment (Karm) [reply src/main/resources/files/liberty/TestSecureController.java.tpl]: Indeed. 
I have no idea :)    @Emily-Jiang ?

Comment (Emily-Jiang) [reply src/main/resources/files/liberty/TestSecureController.java.tpl]: Leave it for now. I'll do a PR to properly tidy up the pom.xml and hope to set context root to /. I will then remove the duplication.

**Pull request merged**